src/macaron/slsa_analyzer/package_registry/deps_dev.py

```diff
@@ -20,28 +22,24 @@ class DepsDevService:
     """The deps.dev service class."""
 
     @staticmethod
-    def get_package_info(purl: str) -> dict | None:
-        """Check if the package identified by the PackageURL (PURL) exists and return its information.
+    def get_endpoint(purl: bool = True, path: str | None = None) -> Any:
```

Comment:
Is there any reason why we have to use `Any` here instead of `urllib.parse.SplitResult`